Using-the-right-Java.md

```diff
@@ -3,78 +3,77 @@ If you don't know which one and how to get it, read on. After you installed the
 
 # Minecraft 1.17 and newer
 
-For Minecraft 1.17 you need to use at least Java 16, for 1.18 you need to use Java 17 so it's easiest to just install Java 17 for both.
```

Comment:
Notice that the earlier version of Forge for Minecraft 1.17.1 is not compatible with Java 17 because of Nashorn, so I personally still recommend to use Java 16 for 1.17.x

Reply:
My thoughts on this is that's perfectly fine. Basically nobody has a reason to use old versions of Forge on that specific MC version, and even if they do run into issues, while providing support in the Discord it's as easy as saying "try updating Forge to recommended. see if that helps".